examples/static-site/build/about/index.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```

Comment:
Would be good to add `examples/static-site/build` to `.gitignore`.

Reply:
I recommend a separate repo of examples, and then either linking to it, or adding as a submodule.